packages/core/test/overflow-list/overflowListTests.tsx

```diff
@@ -238,7 +238,7 @@ describe("<OverflowList>", function (this) {
 
         /** Assert ordered IDs of overflow items. */
         wrapper.assertOverflowItems = (...ids: number[]) => {
-            const overflowItems = wrapper.find(TestOverflow).prop("items");
+            const overflowItems: TestItemProps[] = wrapper.find(TestOverflow).prop("items");
```

Comment:
Out of curiosity, what is the default type here?

Reply:
On `develop`, the default type is inferred as `TestItemProps[]` (as the code would lead you to expect).

However, in the React 18 upgrade branch, the type for `overflowItems` evaluates to `any`. I'm assuming this is an issue with the React 18 enzyme wrapper that we're using to support React 18 in these tests.

![Screenshot 2025-01-14 at 14 39 49@2x](https://github.com/user-attachments/assets/e747a96b-e786-43a3-a69e-74a1da80e2fa)

![Screenshot 2025-01-14 at 14 40 10@2x](https://github.com/user-attachments/assets/8124bef3-8944-450d-a07e-54e8494bfbee)

I'll add an inline code comment to reference this one.